### ISSUE TITLE
for new core branch 

### ISSUE THREAD

**Issue by dhusemann** [dhusemann/idlecombos#10]
;need to adjust this to account for 3 bginstance, count from 0-3, not sure values of ActiveInstance
;ActiveInstance := UserDetails.details.active_game_instance_id, probably a switch based 
If (ActiveInstance == 1) {
	bginstance := 2
}
else
   bginstance :=1 
bginstance ;=0  is the foreground party.

**Comment by Fmagdi** [comment]
I was eager to see the info for the 4th adventure branch and new core in the app so I made modifications to include them.
Changed sections are marked between ";fmagdi -strt" and ";fmagdi -stop" or simply tailed with ";fmagdi" when just one line is changed. Have a look and feel free to include in the next release if useful.
I hope this helps. 

[IdleCombos-mod.txt](https://github.com/dhusemann/idlecombos/files/8902121/IdleCombos-mod.txt)
